env_aop.yaml

```diff
@@ -20,3 +20,4 @@ resource_registry:
   OOShift::LoadBalancer: loadbalancer_neutron.yaml
   OOShift::ContainerPort: sdn_openshift_sdn.yaml
   OOShift::IPFailover: ipfailover_keepalived.yaml
+  OOShift::DockerVolume: volume_docker.yaml
```

Comment:
could you please add also mapping for DockerVolumeAttachment? + document this setting in README